x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/expression/function/EsqlFunctionRegistry.java

```diff
@@ -346,6 +347,7 @@ private static FunctionDefinition[][] functions() {
             // spatial
             new FunctionDefinition[] {
                 def(SpatialCentroid.class, SpatialCentroid::new, "st_centroid_agg"),
+                def(SpatialStExtent.class, SpatialStExtent::new, "st_extent"),
```

Comment:
As much as possible, these should be listed alphabetically.

Reply:
Done.